src/Admin/Customizer/OptinForm/CustomizerControls.php

```diff
@@ -940,6 +940,35 @@ public function configuration_controls()
                         )
                     )
                 ),
+                'optin_sound' => apply_filters('mo_optin_form_customizer_optin_sound_args', array(
+                        'type'        => 'select',
+                        'label'       => __('Optin Sound', 'mailoptin'),
+                        'section'     => $this->customizerClassInstance->configuration_section_id,
+                        'settings'    => $this->option_prefix . '[optin_sound]',
+                        'description' => __('Select the optin sound to play when Optin is displayed', 'mailoptin'),
+                        'choices'     => [
+                            'none'          => __('None', 'mailoptin'),
+                            'beep.wav'          => __('Beep', 'mailoptin'),
+                            'beep-up.wav'       => __('Beep Up', 'mailoptin'),
+                            'beep-down.wav'     => __('Beep Down', 'mailoptin'),
+                            'clong.wav'         => __('Clong', 'mailoptin'),
+                            'chime.aiff'         => __('Chime', 'mailoptin'),
```

Comment:
Chime sound isn't working.

Reply:
If you can get Chime to work, maybe replace it with "Pop" => https://freesound.org/people/elmasmalo1/sounds/377017/

And if you can get it to work, add pop regardless.